org.dash.dash-core.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id": "org.dash.dash-core",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "24.08",
+    "command": "dash-entrypoint.sh",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=~/.dashcore:create",
+        "--filesystem=/run/tor/control.authcookie:ro"
+    ],
+    "modules": [
+        {
+            "name": "dash-core",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -m 0744 -D -t /app/bin/internal bin/dashd",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-qt",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-cli",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-tx",
+                "install -m 0744 -D -t /app/bin/internal bin/dash-wallet",
+                "install -m 0744 -D -t /app/bin/internal bin/test_dash"
+            ],
+            "post-install": [
+                "/app/bin/internal/test_dash"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/dashpay/dash/releases/download/v21.1.1/dashcore-21.1.1-x86_64-linux-gnu.tar.gz",
```

Comment:
can this be built from source?

Reply:
It **can**, however, we use guix for deterministic builds for production see: https://github.com/dashpay/dash/blob/develop/contrib/guix/README.md and https://github.com/dashpay/guix.sigs. Multiple developers all build the exact same source and depends and produce the exact same bit for bit binaries. I think it is more valuable to use the deterministic binary. 

I guess it may be possible for you to use guix in your building, but guix takes a LONG time, and also, I've found it to be flaky in CI like environments